webapp/src/components/common/button/button.tsx

```diff
@@ -12,8 +12,10 @@ export type Props = {
     danger?: boolean;
     iconClass?: string;
     iconPlacement?: 'left' | 'right';
+    disabled?: boolean;
```

Comment:
Great idea! I didn't know about that! Although I had to use `React.ButtonHTMLAttributes<HTMLButtonElement>` instead to give the correct button attributes.

Reply:
I hate that the built-in types have weird, slightly different versions of props like that. I guess that's just something you have to deal since all these types are old